reenact/static/js/scenarios.js

```diff
@@ -1,14 +1,10 @@
-
 function updateScenario(title, description, button) {
-    // Alle Buttons deselektieren
     document.querySelectorAll('#scenarioTabs button').forEach(btn => {
         btn.classList.remove("selected");
     });
 
-    // Gewählten Button markieren
     button.classList.add("selected");
 
-    // Titel & Beschreibung aktualisieren
     document.getElementById("scenarioTitle").textContent = title;
```

Comment:
I think plain texts are not enough here. Instead we need HTML plus styling there.
This can be done using content templates from HTML, see: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template
With this, you can simply write the scenario title and description into the HTML of scenario.html file as templates with id related to scenario. In JS function `updateScenario` you can now load template and replace scenario div with its contents. Could you set this up here?

Reply:
See example here with JS code on how to load and replace div content with template content:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template#examples